doc/api/os.markdown

```diff
@@ -182,5 +182,20 @@ on OS X and `'Windows_NT'` on Windows.
 
 Returns the system uptime in seconds.
 
+## os.userInfo([options])
+
+* `options` {Object}
+  * `encoding` {String} Character encoding used to interpret results.
+    (Default: 'utf8')
+
+Returns a subset of the password file entry for the current effective user. The
+returned object includes the `username`, `uid`, `gid`, `shell`, and `homedir`.
+On Windows, the `uid` and `gid` fields are `-1`, and `shell` is `null`.
+
+The value of `homedir` returned by `userInfo()` comes directly from the
+operating system. This differs from the result of `os.homedir()`, which queries
+several environment variables for the home directory before falling back to the
+operating system response.
+
```

Comment:
nit: perhaps add a short note that if `encoding` is passed as `'buffer'` the `username`, `homedir`, and `shell` will be returned as `Buffer` instances.